products/extensions/b2b-components/employee-management/concept/api-route-restriction-for-employees.md

```diff
@@ -0,0 +1,31 @@
+# API Route Restriction for Employees
+
+## General
+
+B2B employees and Business Partners share the same customer account.
+This can lead to inconsistency for all users of the shared account because they are allowed to change settings and data (both via Storefront and Store-API),
+which are not related to the B2B permission system.
+We decided to restrict most of the customer account routes by implementing a deny-list pattern to prevent the illegal modification of customer data and settings,
+instead of replicating all customer features for employee accounts. We don't consider those features relevant to employee account users.
+
+## Deny-List
+
+The deny-list can be found in the `Resources\config\employee_route_access.xml` config in the employee management folder.
+All denied routes are inside the `<denied>` tags.
+The routes inside the `<allowed>` tags are not important for third party developers because they are used for our internal integration tests to remind our developers to extend the list if new Store-API account routes were added.
```

Comment:
```suggestion
The routes inside the `<allowed>` tags are not important for third party developers, because they are used for internal integration tests to remind developers to extend the list, if new Store-API account routes were added.
```